control-plane-operator/controllers/hostedcontrolplane/hostedcontrolplane_controller.go

```diff
@@ -1218,6 +1221,241 @@ func (r *HostedControlPlaneReconciler) reconcile(ctx context.Context, hostedCont
 		return fmt.Errorf("failed to reconcile default security group")
 	}
 
+	// TODO: consider using a side container in the etcd pod instead.
+	r.Log.Info("Reconciling etcd-backup cronJob")
+	if hostedControlPlane.Spec.Platform.Type == hyperv1.AWSPlatform {
+		configMapName := "etcd-backup-config" // TODO: get configMap name from annotation?
+		configMap := &corev1.ConfigMap{}
```

Comment:
nit: (but can be done in a followup) We should use a manifest function to return the ConfigMap resource.